src/recommendation/index.ts

```diff
@@ -18,3 +18,13 @@ export async function validateAndRecommendExtension(extName: string, message: st
 
   return false;
 }
+
+export async function validateExtensionInstalled(extName: string, version: string) {
+  if(isExtensionInstalled(extName)) {
```

Comment:
if (!isExtensionInstalled(extName)) {
  return false;
}
...

Reply:
updated, actually the in the `validateAndRecommendExtension` take the same check logic